libs/blocks/quiz/quizcontainer.js

```diff
@@ -1,11 +1,6 @@
 import { html } from '../../deps/htm-preact.js';
 
-export const DecorateBlockBackgroundCmp = ({ background = '' }) => html`<picture>
-        <source type="image/webp" srcset=${background} media="(min-width: 400px)" />
-        <source type="image/png" srcset=${background}  media="(min-width: 400px)" />
-        <source type="image/webp" srcset=${background} />
-        <img loading="eager" alt="" type="image/png" src=${background}  width="750" height="375" />
```

Comment:
Great to have these removed, since they were all using the same resource anyway 👍